datanode/gateway/graphql/schema.graphql

```diff
@@ -131,10 +131,8 @@ type Subscription {
 
   "Subscribe to the trades updates"
   trades(
-    "ID of the market from which you want trades updates"
-    marketId: ID
-    "ID of the party from which you want trades updates"
-    partyId: ID
+    "Filter to apply to trades"
+    filter: TradesSubscriptionFilter!
```

Comment:
Can make this not a breaking change? Deprecate the current trades sub and add a new `tradesStream`?

Reply:
Sure, we can do that, it shouldn't be a problem. I'll make the changes.

Reply:
resolved with https://github.com/vegaprotocol/vega/pull/7979/commits/31ba0e6701ea3bd654043a339bf372008810be3f